test/cpp/test_status.cpp

```diff
@@ -0,0 +1,117 @@
+#include <gtest/gtest.h>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "torch_xla/csrc/status.h"
+
+namespace torch_xla {
+
+TEST(StatusTest, MaybeWithLocationRetunsSameStatus) {
```

Comment:
Append `WhenContextIsDisabled` to the name.

Reply:
See [this comment][1].

[1]: https://github.com/pytorch/xla/pull/9386#discussion_r2173269231